### ISSUE TITLE
Citability

### ISSUE THREAD

**Issue by PonteIneptique** [perseids-publications/treebank-template#23]
Dear @zfletch ,
I think it would be great to hook the repositories here to zenodo.org :) I was gonna try to cite the different corpora in a paper, but there is no longterm solution available, nor real bibliography information. Using release versioning + zenodo hook + metadata, you'd be able to convey nice informations + DOI which would definitely be nice :)

**Comment by zfletch** [comment]
That's a great idea and it looks like it will be fairly easy to do thanks to the GitHub integration. The main challenge will be how to modify the UI to include the DOI. I will create a branch and start trying some different options.

**Comment by PonteIneptique** [comment]
Yeah, the hard part are definitely not getting the DOI :) Thanks !

**Comment by zfletch** [comment]
I've added Zenodo instructions to the README in [v3.0](https://github.com/perseids-publications/treebank-template/tree/v3.0.0). It is more complicated than I thought it would be due to the fact that we need the Zenodo DOI *before* publication in order to display it. But this system should work for now. We will eventually update it to be automatic when the Zenodo GitHub integration supports the necessary features.

As a proof of concept, I created a [DOI](https://zenodo.org/record/3827931) for the Treebank Template software resource itself (though some of the metadata is still subject to change). I will discuss with the contributors of the other repositories and roll out the DOI functionality over the next few weeks.

Thanks again for the suggestion @PonteIneptique!
